website/docs/client/05-plugins/01-standard-plugins.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Standard plugins
+---
+
+# Standard plugins
+
+Ariadne Codegen ships with optional plugins importable from the `ariadne_codegen.contrib` package:
+
+- [`ariadne_codegen.contrib.shorter_results.ShorterResultsPlugin`](ariadne_codegen/contrib/shorter_results.py) - This plugin processes generated client methods for operations where only single top field is requested, so they return this field's value directly instead of operation's result type. For example get_user method generated for query `GetUser() { user(...) { ... }}` will return value of user field directly instead of `GetUserResult`.
```

Comment:
This and other links result in a 404 page. It should be like that:

```suggestion
- [`ariadne_codegen.contrib.shorter_results.ShorterResultsPlugin`](https://github.com/mirumee/ariadne-codegen/blob/main/ariadne_codegen/contrib/shorter_results.py) - This plugin processes generated client methods for operations where only single top field is requested, so they return this field's value directly instead of operation's result type. For example get_user method generated for query `GetUser() { user(...) { ... }}` will return value of user field directly instead of `GetUserResult`.
```

If you want to refer to github repo